<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterToPdfDialog</class>
 <widget class="QDialog" name="RegisterToPdfDialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>1</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
    <pointsize>10</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Loading layers from online resources</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/plugins/GIAP-PolaMap/icons/giap_logo.png</normaloff>:/plugins/GIAP-PolaMap/icons/giap_logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
background-color: #365871;
color:#E0DECF;
font: 10pt &quot;Segoe UI&quot;;
}
QLabel{
font: 10pt &quot;Segoe UI&quot;;
color:#E0DECF;
height: 35px;
}
QCheckBox::indicator {
  width: 13px;
    height: 13px;
border: 0px solid #E0DECF;
}
QCheckBox::indicator:checked {
 image: url(:/plugins/GIAP-PolaMap/icons/checked.png);
  width: 13px;
    height: 13px;
}
QCheckBox::indicator:unchecked {
 image: url(:/plugins/GIAP-PolaMap/icons/unchecked.png);
  width: 13px;
    height: 13px;
}

QScrollBar:horizontal {
    height: 15px;
}
QScrollBar::handle:horizontal {
    background: #5589B0;
    border-radius: 5px;
    width: 10px;
    margin: 2px 0 2px 0;
}
QScrollBar::add-line:horizontal, QScrollBar::sub-line:horizontal  {
    border: 1px solid transparent;
    background: transparent;
    width: 12px;
}
 QScrollBar:vertical {
     width: 15px;
 }
 QScrollBar::handle:vertical {
     background: #5589B0;
     height: 10px;
     border-radius: 5px;
         margin: 0px 2px 0px 2px;
 }
 QScrollBar::add-line:vertical{
    border: 1px solid transparent;
     background: transparent;
     height: 12px;
 }

 QScrollBar::sub-line:vertical {
      border: 1px solid transparent;
     background: transparent;
     height: 12px;
 }

QAbstractScrollArea::corner {
 border: 1px solid transparent;
width: 0px; 
height: 0px;
}

QPushButton {
padding: 5px;
}

QLineEdit, QTextEdit{
border: 1px solid;
border-radius:3px;
border-color: #263c4e;
background-color: #263c4e;
}

QLineEdit:focus{
border: solid;
border-style: solid;
border-width: 1px;
border-color: #E0DECF;
}

QPushButton {
border: none;
border-width: 1px;
border-radius:3px;
background-color: #5589B0;
}
QPushButton:checked {
background-color: #5589B0;
border: solid;
border-width: 1px;
border-color: #5589B0;
}

QPushButton:pressed {
background-color: #5589B0;
border: solid;
border-width: 1px;
border-color:#5589B0;
}
QPushButton:hover {
background-color: #5589B0;
border-radius: 3px;
border: solid;
border-style: solid;
border-width: 1px;
border-color: #E0DECF;
}

QToolButton {
border: none;
border-width: 1px;
border-radius:3px;
background-color: #5589B0;
}

QToolButton:checked {
background-color: #5589B0;
border: solid;
border-width: 1px;
border-color: #5589B0;
}
QToolButton:hover {
background-color: #5589B0;
border-radius: 3px;
border: solid;
border-style: solid;
border-width: 1px;
border-color: #E0DECF;
}


QToolButton:pressed {
background-color: #5589B0;
border: solid;
border-width: 1px;
border-color:#5589B0;
}

QToolButton::menu, QToolButton::menu-button {
background-color: #263C4E;
border: solid;
border-width: 1px;
border-color: #5589B0;
}

QMenu::item:selected {
background-color:  #5589B0;
}

QComboBox QListView{
selection-background-color:  #5589B0;
    color: #E0DECF;
     padding: 2px;
}

QComboBox:editable {
border: 0px solid #5589B0;
border-radius: 3px;
background-color: #263c4e;
padding: 0px 0px 0px 3px;
}

QComboBox:editable:focus{
border: solid;
border-style: solid;
border-width: 1px;
border-color: #E0DECF;
}

QComboBox:!editable {
border: 0px solid #5589B0;
border-radius: 3px;
background-color: #263c4e;
padding:0px 3px 0px 3px;
}

QComboBox:!editable:on {
background-color: #263c4e;
border: solid;
border-width: 1px;
border-color: #365871;
padding:0px 5px 0px 3px;
}

QComboBox::drop-down {
border-left: solid;
border-width: 0px;
background-color: #263c4e;
border-radius: 3px;
}
QComboBox::drop-down:editable:on {
background-color:#263c4e;
border: solid;
border-width: 1px;
border-color: #263c4e;
}

QComboBox::drop-down:!editable:on {
background-color: #263c4e;
}

QComboBox::down-arrow {
        image:  url(:/plugins/GIAP-PolaMap/icons/branch-open.png);
    width: 18px;
    height: 28px;
}

QComboBox::down-arrow:hover, QComboBox::down-arrow:pressed{
border: solid;
border-width: 0px;
border-color: #E0DECF;
border-radius: 3px;
margin:0px 3px 0px -1px;
}

QToolTip {
color:white;
	font: 9pt &quot;Segoe UI&quot;;
font-weight: normal;
border-style: double;
border: 0.1rem;
border-radius: 0.3rem;
background-color:#4F5A63;
}

Line {
color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(65, 97, 124, 255), stop:1 rgba(90, 135, 172, 255));
}

QTabBar::tab {
border-top-left-radius: 3px;
border-top-right-radius: 3px;
padding:1px 0px 1px 0px;
margin-top: 2px;
color: #E0DECF;
font-weight: 550;
}

QTabBar::tab:selected {
border: 0rem solid;
border-color:#5589B0;
margin-bottom: 2px;
border-bottom: 2px solid #E0DECF;
font-weight: 650;
color: #E0DECF;
}
QTabBar::tab:hover {
border: 0rem solid;
background: transparent;
margin-bottom: 2px;
border-bottom: 2px solid #E0DECF;
font-weight: 650;
}

QTabBar::tab:!selected {
border: 0rem solid;
background: transparent;
margin-bottom: 2px;
border-bottom: 2px solid transparent;
font-weight: 550;
color: #B3B2A6;
}
QTabBar::tab:!selected:hover {
border: 0rem solid;
background: transparent;
margin-bottom: 2px;
border-bottom: 2px solid #E0DECF;
font-weight: 550;
color: #B3B2A6;
}

QTabBar::tab:last:!selected {
margin-right: 1px;
}

QTabWidget::pane {
border-top:1px solid #B3B2A6;
border-bottom: 0px solid #B3B2A6;
}

QTabWidget::tab-bar {
left: 5px;
right: 5px;
}

QTreeView {
    border: 0rem solid;
    border-color: #5589B0;
    margin: 0.2rem;
   show-decoration-selected: 1;
}

QTreeView::item,
QTreeView::branch {
    color: black;
     padding:2px;
}

QTreeView:branch:hover,
QTreeView:item:hover
{
    background-color: #5589B0;
    color: white;
}

QTreeView:branch:selected,
QTreeView:item:selected
{
    background-color: #5589B0;
    color: white;
}

QTreeView::branch:has-children:!has-siblings:closed,
QTreeView::branch:closed:has-children:has-siblings {
        border-image: none;
        image: url(:/plugins/GIAP-PolaMap/icons/branch-closed.png);
}

QTreeView::branch:open:has-children:!has-siblings,
QTreeView::branch:open:has-children:has-siblings  {
        border-image: none;
        image:  url(:/plugins/GIAP-PolaMap/icons/branch-open.png);
}

QProgressBar {
    border: 0.2rem solid #365871;
    background-color: #365871;
    border-radius: 0.3rem;
    text-align: center;
    color: white;
}

QProgressBar::chunk {
    background-color: #5589B0;
}
QSpinBox, QDoubleSpinBox,
QSpinBox:editable,
QSpinBox:hover,
QSpinBox:pressed,
QDoubleSpinBox:editable,
QDoubleSpinBox:hover,
QDoubleSpinBox:pressed {
    border: 0.1rem solid;
    border-radius: 0.3rem;
    border-color: #263c4e;
background-color: #263c4e;
}
QSpinBox:focus, QDoubleSpinBox:focus {
border-style: solid;
border-width: 0rem;
border-color: #E0DECF;
}
QSpinBox:disabled, QDoubleSpinBox:disabled {
  border: none;
  border-bottom: 0.1rem solid #263c4e;
  border-radius: 0;
  background: #263c4e;
  color: #000000;
}
QSpinBox::up-button, QDoubleSpinBox::up-button{
  subcontrol-origin: border;
  subcontrol-position: top right;
  border: 0rem solid white;
  border-radius: 3;
  background: #263c4e;
}
QSpinBox::down-button, QDoubleSpinBox::down-button {
  subcontrol-origin: border;
  subcontrol-position: bottom right;
  border: 0rem solid white;
  border-radius: 3;
  background: #263c4e;
}
QSpinBox::down-button:hover, QDoubleSpinBox::down-button:hover, QSpinBox::up-button:hover, QDoubleSpinBox::up-button:hover {
border: solid;
border-width: 0.1rem;
border-color: #E0DECF;
}
QSpinBox::up-arrow,QDoubleSpinBox::up-arrow {
  image: url(:/plugins/GIAP-PolaMap/icons/spinup.png);
    width: 18px;
    height: 18px;
}
QSpinBox::up-arrow:disabled,
QSpinBox::up-arrow:off {
  image: none;
}
QSpinBox::down-arrow, QDoubleSpinBox::down-arrow {
  image: url(:/plugins/GIAP-PolaMap/icons/spindown.png);
    width: 18px;
    height: 18px;
}
QSpinBox::down-arrow:disabled,
QSpinBox::down-arrow:off,
 QDoubleSpinBox::down-arrow:disabled,
QDoubleSpinBox::down-arrow:off{
  image: none;
}

QTableView {
border:1px solid #2A4457;
color:white;
selection-background-color: #6FB1E3;
 selection-color: white;
}
QTableView::item:selected {
color:white;
selection-background-color: #6FB1E3;
selection-color: white;
}</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="1">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="zakoncz">
      <layout class="QGridLayout" name="gridLayout_15">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0" colspan="2">
        <widget class="QFrame" name="frame_10">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #4f5a63;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <property name="leftMargin">
           <number>9</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="title_label_9">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Segoe UI</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color : #EDF6FC;	font-weight: bold;</string>
            </property>
            <property name="text">
             <string>Loading layers from online resources</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" rowspan="2" colspan="2">
        <widget class="QFrame" name="frame_11">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="1" column="0">
           <widget class="QFrame" name="frame_7">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_3">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <item row="1" column="3">
              <widget class="QPushButton" name="save_btn">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>170</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>Load</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0" colspan="2">
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="1" column="2">
              <widget class="QPushButton" name="select_btn">
               <property name="minimumSize">
                <size>
                 <width>170</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>Select/Unselect all</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0" colspan="5">
              <widget class="QListView" name="layers_checklist">
               <property name="styleSheet">
                <string notr="true">QTableView, QListView
        {
			background-color: rgb(240, 240, 240);
            gridline-color:#4F5A63;
            color: #263c4e;
			alternate-background-color:  rgb(240, 240, 240);
			alternate-color: #263c4e;
            border: 1px solid #5589B0;
        }
QTableView::item, QListView::item
        {
            color: #263c4e;
        }
QTableView::item:hover, QListView::item:hover
        {
            color: white;
            background: #5589B0;
            selection-background-color: #5589B0;
        }
QTableView::item:focus, QListView::item:focus
        {
            color: white;
            background: #263B4E;
        }
QTableView::item:selected, QListView::item:selected
        {
            color: white;
            background-color: #5589B0;
        }
</string>
               </property>
               <property name="editTriggers">
                <set>QAbstractItemView::NoEditTriggers</set>
               </property>
               <property name="selectionMode">
                <enum>QAbstractItemView::SingleSelection</enum>
               </property>
               <property name="selectionBehavior">
                <enum>QAbstractItemView::SelectRows</enum>
               </property>
              </widget>
             </item>
             <item row="1" column="4">
              <widget class="QPushButton" name="cancel_btn">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>170</width>
                 <height>0</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Segoe UI</family>
                 <pointsize>10</pointsize>
                 <weight>50</weight>
                 <italic>false</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="text">
                <string>Cancel</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QFrame" name="frame_2">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_5">
             <item row="0" column="2">
              <widget class="QPushButton" name="clear_search_btn">
               <property name="maximumSize">
                <size>
                 <width>40</width>
                 <height>23</height>
                </size>
               </property>
               <property name="text">
                <string>X</string>
               </property>
               <property name="autoDefault">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="lineEdit">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>23</height>
                </size>
               </property>
               <property name="placeholderText">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="0" rowspan="2">
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
          * {
          color: #0A0C0D;
          }
          QFrame {
          border-right: 2px solid #b7d3e8;
          }
      </string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <property name="leftMargin">
       <number>12</number>
      </property>
      <property name="topMargin">
       <number>12</number>
      </property>
      <property name="rightMargin">
       <number>12</number>
      </property>
      <property name="verticalSpacing">
       <number>9</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="radioButton_1">
        <property name="styleSheet">
         <string notr="true">* {
background-color: #365871;
color:#E0DECF;
font: 10pt &quot;Segoe UI&quot;;
}
QLabel{
font: 10pt &quot;Segoe UI&quot;;
color:#E0DECF;
height: 35px;
}</string>
        </property>
        <property name="text">
         <string>Loading layers from
online resources</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
     <zorder>verticalSpacer_3</zorder>
     <zorder>radioButton_1</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancel_btn</sender>
   <signal>clicked()</signal>
   <receiver>RegisterToPdfDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>740</x>
     <y>544</y>
    </hint>
    <hint type="destinationlabel">
     <x>429</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clear_search_btn</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>809</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>489</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
